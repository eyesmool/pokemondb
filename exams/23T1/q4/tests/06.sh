psql bank -c 'select q4(654167);'
