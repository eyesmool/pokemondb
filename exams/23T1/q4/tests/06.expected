                           q4                           
--------------------------------------------------------
 Mismatch: calculated balance 1400, stored balance 1000
(1 row)
